spec/models/game_spec.rb

```diff
@@ -1,7 +1,39 @@
 require 'rails_helper'
 
 RSpec.describe Game, type: :model do
-  describe 'populate game' do
+  describe '#available' do
+    context 'When there is no game at all' do
+      it 'Should return an empty array' do
+        expect(Game.available).to eq []
+      end
+    end
+
+    context 'When there are games' do
+      let(:game) { FactoryGirl.create(:game) }
+
+      context 'When all games have both players' do
+        it 'Should return an empty array' do
+          expect(Game.available).to eq []
```

Comment:
Since `let` is `lazy`, the `game` won't exist at the time this test is called, because `game` isn't explicitly called.

If you change `let` to `let!` then it will be created.